src/core/freebsd/SDL_evdev_kbd_freebsd.c

```diff
@@ -292,9 +298,14 @@ SDL_EVDEV_kbd_init(void)
 void
 SDL_EVDEV_kbd_quit(SDL_EVDEV_keyboard_state *kbd)
 {
+    struct mouse_info mData;
+
     if (!kbd) {
         return;
     }
+    memset(&mData, 0, sizeof(struct mouse_data));
```

Comment:
Make this an `SDL_zero(mData)`, too, please. I assume that "sizeof `mouse_data` was meant to be `mouse_info`, like the others, and the SDL_zero macro would have prevented that.